docs/rules/array-element-newline.md

```diff
@@ -0,0 +1,242 @@
+# enforce line breaks after each array element (array-element-newline)
+
+A number of style guides require or disallow line breaks after each array element.
+
+## Rule Details
+
+This rule enforces line breaks after each array element.
+
+## Options
+
+This rule has either a string option:
+
+* `"always"` (default) requires line breaks inside braces
+* `"never"` disallows line breaks inside braces
+
+Or an object option (Requires line breaks if any of properties is satisfied. Otherwise, disallows line breaks):
+
+* `"multiline": <boolean>` requires line breaks if there are line breaks inside elements or between elements. If this is false, this condition is disabled.
```

Comment:
Am I correct that the original behavior, from @mysticatea, is not valid anymore (https://github.com/eslint/eslint/issues/6075 - very first issue description)?

Me, as a user, would think that `multiline` will enforce this rule, as soon as there is a linebreak (as currently implemented)